docs/src/main/paradox/events-by-tag.md

```diff
@@ -8,100 +8,112 @@ Due to it being a separate table and events are written from multiple nodes the
+As events come from multiple nodes to deliver events in `TimeUUID` order there is an `eventual-consistency-delay` to give
+events time to arrive in Cassandra and for the query to deliver them in their final order. The `eventual-consistency-delay` keeps the query a configurable 
+amount of time in the past.
 
-This is mitigated by an additional sequence number per persistence id/tag combination that is used to detect if an event
-has been missed. Upon detection the query will search back in time for the missed events.
+The events by tag stream updates the Cassandra query it uses to start from the latest offset to avoid reading events it has already delivered. 
+If events are delayed and the query progresses past an event's offset it will be detected either when the next event for that 
+persistence id is received or if no more events are received for the persistence id then it will be detected via [back tracking](#back-tracking). 
 
+In both cases the query will go back and search for the delayed events. The search for missing events is an expensive operation
+so it is advised not to set the `eventual-consistency-delay` too low as it increased the liklihood of missed events.
 
-To detect missed events without receiving another event for the same persistence id a periodic back track is done
-to find events. This is configured with:  
+## Back tracking
 
-```
-cassandra-query-journal.events-by-tag.back-track {
-      # Interval at which events by tag stages trigger a query to look for delayed events from before the
-      # current offset. Delayed events are also searched for when a subsequent event for the same persistence id
-      # is received. The purpose of this back track is to find delayed events for persistence ids that are no
-      # longer receiving events. Depending on the size of the period of the back track this can be expensive.
-      interval = 1s
-
-      # How far back to go in time for the scan. This is the maxinum amount of time an event can be delayed
-      # and be found without receiving a new event for the same persistence id. Set to a duration or "max" to search back
-      # to the start of the previous bucket which is the furthest a search can go back.
-      period = 5s
-}
-```
+If another event for the same persistence id is received the sequence number is used to detect that an event has been missed.
+To detect missed events without receiving another event for the same persistence id a periodic backtrack is done
+to find events. Queries are run in the background to detect delayed events. A high frequency short back track is done
+for finding events delayed a small amount and a low frequency backtrack that scans further back. 
+
+These are configured with:
 
+@@snip [refernce.conf](/core/src/main/resources/reference.conf) { #backtrack }                                                                                                                                
```

Comment:
in the included snippet it will not show the full config path: `cassandra-query-journal.events-by-tag.back-track`.
Maybe just clarify that with

These are configured with `cassandra-query-journal.events-by-tag.back-track`: